src/sections/transaction/ReviewTransactionAuthorTip.tsx

```diff
@@ -67,23 +69,39 @@ export const ReviewTransactionAuthorTip: FC<Props> = ({
   )
 
   return (
-    <SContainer>
-      <SInputContainer>
-        <SInput
-          placeholder={t("liquidity.reviewTransaction.modal.amount")}
-          value={amount}
-          onChange={(e) => setAmount(e.target.value)}
-        />
-        <Text color="darkBlue200">{asset.symbol}</Text>
-      </SInputContainer>
-      <Text
-        tAlign="right"
-        fs={12}
-        lh={16}
-        color={error ? "red400" : "darkBlue300"}
-      >
-        {error ? error : <DisplayValue value={displayValue} />}
-      </Text>
-    </SContainer>
+    <div sx={{ flex: "column", gap: 8 }}>
+      <Switch
+        size="small"
+        value={visible}
+        onCheckedChange={(value) => {
+          setVisible(value)
+
+          if (!value) setAmount("")
+        }}
+        name={"liquidity.reviewTransaction.modal.detail.tip.name"}
+        css={{ justifyContent: "end" }}
```

Comment:
```suggestion
        sx={{ ml: "auto" }}
```

this should also work with same result

Reply:
yes, but a different solution